commit 918b9e451bbaebb775725a292fd3b6e3d4cf804f
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Fri Dec 5 17:26:26 2014 +0000

    fdt: Add support for the CONFIG_CMDLINE_EXTEND option

diff --git a/drivers/of/fdt.c b/drivers/of/fdt.c
index 655f79d..fdc4501 100644
--- a/drivers/of/fdt.c
+++ b/drivers/of/fdt.c
@@ -954,19 +954,38 @@ int __init early_init_dt_scan_chosen(unsigned long node, const char *uname,
 
 	/* Retrieve command line */
 	p = of_get_flat_dt_prop(node, "bootargs", &l);
-	if (p != NULL && l > 0)
-		strlcpy(data, p, min((int)l, COMMAND_LINE_SIZE));
 
 	/*
 	 * CONFIG_CMDLINE is meant to be a default in case nothing else
 	 * managed to set the command line, unless CONFIG_CMDLINE_FORCE
 	 * is set in which case we override whatever was found earlier.
+	 *
+	 * However, it can be useful to be able to treat the default as
+	 * a starting point to be extended using CONFIG_CMDLINE_EXTEND.
 	 */
+	((char *)data)[0] = '\0';
+
 #ifdef CONFIG_CMDLINE
-#ifndef CONFIG_CMDLINE_FORCE
-	if (!((char *)data)[0])
+	strlcpy(data, CONFIG_CMDLINE, COMMAND_LINE_SIZE);
+
+	if (p != NULL && l > 0)	{
+#if defined(CONFIG_CMDLINE_EXTEND)
+		int len = strlen(data);
+		if (len > 0) {
+			strlcat(data, " ", COMMAND_LINE_SIZE);
+			len++;
+		}
+		strlcpy((char *)data + len, p, min((int)l, COMMAND_LINE_SIZE - len));
+#elif defined(CONFIG_CMDLINE_FORCE)
+		pr_warning("Ignoring bootargs property (using the default kernel command line)\n");
+#else
+		/* Neither extend nor force - just override */
+		strlcpy(data, p, min((int)l, COMMAND_LINE_SIZE));
 #endif
-		strlcpy(data, CONFIG_CMDLINE, COMMAND_LINE_SIZE);
+	}
+#else /* CONFIG_CMDLINE */
+	if (p != NULL && l > 0)
+		strlcpy(data, p, min((int)l, COMMAND_LINE_SIZE));
 #endif /* CONFIG_CMDLINE */
 
 	pr_debug("Command line is: %s\n", (char*)data);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1049_918b9e451bbaebb775725a292fd3b6e3d4cf804f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1049_918b9e451bbaebb775725a292fd3b6e3d4cf804f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
